delivery_tnt_oca/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery TNT OCA",
+    "summary": "Integrate TNT webservice",
+    "version": "13.0.1.0.0",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "delivery",
+        "delivery_package_number",
+        "delivery_state",
+        "product_dimension",
+        "base_sparse_field",
```

Comment:
I see you're defining ZPL templates, but I see no dependency on any zpl module. Shouldn't you add it?

Reply:
In my opinion there is no need to add any dependencies relative to .zpl.
Odoo allow us to create a report that has the content of a label in zpl format; this addon is generating that report and creating the attachment with the name "TNT-XX.zpl" to have the file that will later be printed.

Reply:
OK I understand.